### PR TITLE
No need to polyfill anything

### PR DESCRIPTION
**Pull request by i-like-robots**
I don't think there's any need to load the ES5 polyfills. 

Related: https://github.com/Financial-Times/next-user-model/pull/22

^ only to make the tests pass in IE, it should work OK on live.

### CODE REVIEW

**Comment by keirog** [comment]
Good, I was wondering about this addition too.

File: client/js/ftcom.js [hunk 1]
```diff
@@ -14,20 +14,19 @@ var css = requireText('../../sass-build/ftcom.css');
 var html = requireText('../ftcom.html');
 
 function trackEvents() {
-	var hasEventListener = 'addEventListener' in window;
 	var hasTracerMethod = !!(window.FT && FT.analytics && FT.analytics.siteIntelligence && typeof FT.analytics.siteIntelligence.sendAdditionalTracer === 'function');
 	var link = document.querySelector('.next-opt__link');
 
 	// We're not showing the link to <IE9 anyway, so no attachEvent needed
```
Comment (keirog) [inline client/js/ftcom.js]: Comment no longer true

Comment (i-like-robots) [inline client/js/ftcom.js]: lies removed

**Comment by keirog** [comment]
:+1: for monday

**Comment by i-like-robots** [comment]
Do I need to precompile before merge @keirog ?

**Comment by keirog** [comment]
Nope, no need

**Pull request merged**